<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>359</width>
    <height>424</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>359</width>
    <height>424</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>359</width>
    <height>424</height>
   </size>
  </property>
  <property name="sizeIncrement">
   <size>
    <width>367</width>
    <height>424</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="frame_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>2</y>
      <width>357</width>
      <height>421</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(227, 227, 227);
border-radius: 20px;

border-width: 2px; 
border-radius: 5px;
border-style: outset; 
border-color: rgb(0, 0, 0);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLineEdit" name="lineEdit_5">
     <property name="geometry">
      <rect>
       <x>81</x>
       <y>122</y>
       <width>206</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="placeholderText">
      <string>Digite o Nome do Usuário Desejado.</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_6">
     <property name="geometry">
      <rect>
       <x>81</x>
       <y>188</y>
       <width>203</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="placeholderText">
      <string>Digite sua Senha EX: 2054@#25P</string>
     </property>
    </widget>
    <widget class="QPushButton" name="bt_sairtela03_3">
     <property name="geometry">
      <rect>
       <x>261</x>
       <y>380</y>
       <width>86</width>
       <height>32</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>MS Shell Dlg 2</family>
       <pointsize>12</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
       <kerning>true</kerning>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
background-color: rgb(227, 227, 227);
	color: rgb(51, 0, 255);

border-width: 0px; 
border-radius: 5px;
border-style: outset; 
border-color: rgb(0, 0, 0);
}

QPushButton:pressed{
background-color: rgb(227, 227, 227);
color: rgb(0, 12, 194);
border-width: 0px; 
border-radius: 5px;
border-style: outset; 
border-color: rgb(255, 255, 255);



}
 </string>
     </property>
     <property name="text">
      <string>Voltar</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>imagens Projeto/voltar (1).png</normaloff>imagens Projeto/voltar (1).png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>30</width>
       <height>30</height>
      </size>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_6">
     <property name="geometry">
      <rect>
       <x>81</x>
       <y>320</y>
       <width>205</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
color: rgb(0, 0, 0);
border-width: 2px; 
border-radius: 8px;
background-color: rgb(0, 217, 105);
border-style: outset; 
border-color: rgb(0, 0, 0);
}

QPushButton:pressed{
background-color: rgb(114, 170, 84);
font: 87 10pt &quot;Arial &quot;;
border-width: 2px; 
border-radius: 10px;
border-style: outset; 
border-color: rgb(234, 234, 234);

}
 </string>
     </property>
     <property name="text">
      <string>Finalizar cadastro</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>81</x>
       <y>101</y>
       <width>113</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">
border-width: 0px; 
border-radius: 0px;
border-style: outset; 
border-color: rgb(0, 0, 0);</string>
     </property>
     <property name="text">
      <string>Nome de Usuário : </string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>81</x>
       <y>168</y>
       <width>106</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">
border-width: 0px; 
border-radius: 0px;
border-style: outset; 
border-color: rgb(0, 0, 0);</string>
     </property>
     <property name="text">
      <string>Digite sua senha :</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_7">
     <property name="geometry">
      <rect>
       <x>81</x>
       <y>254</y>
       <width>206</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="placeholderText">
      <string>Digite sua Senha EX: 2054@#25P</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>81</x>
       <y>234</y>
       <width>115</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">
border-width: 0px; 
border-radius: 0px;
border-style: outset; 
border-color: rgb(0, 0, 0);</string>
     </property>
     <property name="text">
      <string>Confirme sua senha : </string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>91</x>
       <y>293</y>
       <width>190</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Bahnschrift</family>
       <pointsize>10</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">
color: rgb(234, 0, 3);
border-width: 0px; 
border-radius: 0px;
border-style: outset; 
border-color: rgb(0, 0, 0);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_8">
     <property name="geometry">
      <rect>
       <x>81</x>
       <y>59</y>
       <width>206</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="placeholderText">
      <string>Digite seu nome.</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>81</x>
       <y>32</y>
       <width>113</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">
border-width: 0px; 
border-radius: 0px;
border-style: outset; 
border-color: rgb(0, 0, 0);</string>
     </property>
     <property name="text">
      <string>Qual o seu nome ? </string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
